lib/mayaUsd/ufe/UsdRotateUndoableCommand.h

```diff
@@ -31,40 +32,45 @@ namespace ufe {
 //! \brief Absolute rotation command of the given prim.
 /*!
 	Ability to perform undo to restore the original rotation value.
-	As of 06/07/2018, redo is a no op as Maya re-does the operation for redo
  */
-class MAYAUSD_CORE_PUBLIC UsdRotateUndoableCommand : public Ufe::RotateUndoableCommand
+class MAYAUSD_CORE_PUBLIC UsdRotateUndoableCommand : public Ufe::RotateUndoableCommand, public UsdTRSUndoableCommandBase<GfVec3f>
```

Comment:
Derive from the helper class that contains all the factored-out, common code for translation, rotate, scale.